src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+.page {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
```

Comment:
it does the job for the current task,
yet @mmahfuzi when need to stick with a design measures - it's better to provide the 
explicit `_px` spacing

Reply:
> it does the job for the current task, yet @mmahfuzi when need to stick with a design measures - it's better to provide the explicit `_px` spacing

Thanks for your comment.
Sure, if I was allowed to use 'gap' I would add explicit px and was easier to handle.

Reply:
@mmahfuzi i meant that you could use `margin-right: 48px` ))